tools/pylint-extensions/azure-pylint-guidelines-checker/tests/test_pylint_custom_plugins.py

```diff
@@ -302,202 +303,131 @@ def test_guidelines_link_active(self):
         assert response.http_response.status_code == 200
 
 
+def _load_file(filename):
```

Comment:
oh nice :)

Reply:
Thank you, it allowed for some of my tests to be simplified.